packages/path_provider/lib/path_provider.dart

```diff
@@ -57,14 +75,12 @@ Future<Directory> getApplicationDocumentsDirectory() async {
 /// On iOS, this function throws an UnsupportedError as it is not possible
 /// to access outside the app's sandbox.
 ///
-/// On Android this returns getExternalStorageDirectory.
+/// On Android this returns the external storage directory.
```

Comment:
I think the previous comment actually had a specific meaning that gets lost by making this plain English, it was referencing https://developer.android.com/reference/android/os/Environment#getExternalStorageDirectory(). Maybe link it here still?